src/components/HexColorInput.tsx

```diff
@@ -53,4 +52,4 @@ const HexInput = (props: Partial<Props>) => {
   return React.createElement("input", inputProps);
 };
 
-export default React.memo<Partial<Props>>(HexInput);
+export const HexColorInput = React.memo<Partial<Props>>(HexColorInputBase);
```

Comment:
Are we planning on using `HexColorInputBase` anywhere else? Could just wrap the main component in the `React.memo`. 

```suggestion
export const HexColorInput = React.memo<Partial<Props>>((...) => { ... });
```

Reply:
Understand what you mean but:
1) I don't like a mess like `React.memo<Partial<Props>>((props: Partial<Props>)`. It seems unreadable to me.
2) These things might throw ES lint warning about missing displayName.